docs/do-not-merge.md

```diff
@@ -53,20 +53,21 @@ For more information about this feature, see the [26.0.0 release notes for Schem
 
 ## Smart segment loading
 
-Revamped Coordinator to make it more stable and user-friendly. This is accompanied by several bug fixes, logging and metric improvements, and a whole new range of capabilities.
+The Coordinator is now more stable and user-friendly. This is accompanied by several bug fixes, logging and metric improvements, and a whole new range of capabilities.
```

Comment:
"The Coordinator" seems to have been repeated several times in this section, can we reduce it?